imports/ui/lists/recentAgents.coffee

```diff
@@ -34,9 +49,59 @@ Template.recentAgents.onCreated ->
         if @recentAgents.find(articleId: article._id).count() > 5
           @articles.update(article._id, { $set: { collapsed: true } })
 
+
+Template.recentAgents.onRendered ->
+  Meteor.startup =>
+    prevScrollPos = window.pageYOffset
+
+    infiniteScroll = (options) ->
+      defaults = {
+        distance: 100
+        callback: (done) -> done()
+      }
+      for key in defaults
```

Comment:
This should be `for key of defaults`, but I would suggest _.extend() instead of a loop.